### PR TITLE
Add load from/to memory from immediate a16

### PR DESCRIPTION
**Pull request by ricfehr3**

### CODE REVIEW

File: src/cpu/mod.rs [hunk 1]
```diff
@@ -805,6 +805,16 @@ impl Cpu {
                 self.ld_from_mem(Register8Bit::A, 0xff00 + (offset as u16));
                 (pc + 2, 3)
             },
+            Instruction::LdToImmMem() => {
+                let addr = self.read_word(pc + 1);
+                self.ld_to_mem(Register8Bit::A, addr);
+                (pc + 3, 2)
```
Comment (toastbucket) [inline src/cpu/mod.rs:811]: cycles should be 4

File: src/cpu/mod.rs [hunk 2]
```diff
@@ -805,6 +805,16 @@ impl Cpu {
...
+            },
+            Instruction::LdFromImmMem() => {
+                let addr = self.read_word(pc + 1);
+                self.ld_from_mem(Register8Bit::A, addr);
+                (pc + 3, 2)
```
Comment (toastbucket) [inline src/cpu/mod.rs:816]: cycles should be 4

**Review by toastbucket — CHANGES_REQUESTED** [review]

File: src/cpu/mod.rs (hunk 1 above)
Comment (ricfehr3) [reply src/cpu/mod.rs:811]: omg

File: src/cpu/mod.rs (hunk 2 above)
Comment (ricfehr3) [reply src/cpu/mod.rs:816]: OH EM GEE

**Comment by toastbucket** [comment]
cool and epic 😎

**Review by toastbucket — APPROVED** [review]

**Pull request merged**